app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelAboutFragment.java

```diff
@@ -26,15 +26,10 @@ public class ChannelAboutFragment extends BaseDescriptionFragment {
     @State
     protected ChannelInfo channelInfo;
 
-    public static ChannelAboutFragment getInstance(final ChannelInfo channelInfo) {
-        final ChannelAboutFragment fragment = new ChannelAboutFragment();
-        fragment.channelInfo = channelInfo;
-        return fragment;
+    ChannelAboutFragment(@NonNull final ChannelInfo channelInfo) {
+        this.channelInfo = channelInfo;
     }
 
-    public ChannelAboutFragment() {
-        super();
-    }
```

Comment:
Actually, the empty constructors are needed when resuming a fragment from memory:

`Caused by: java.lang.NoSuchMethodException: org.schabi.newpipe.fragments.list.channel.ChannelAboutFragment.<init> []`